ethereum/contracts/BeefyLightClient.sol

```diff
@@ -93,6 +101,8 @@ contract BeefyLightClient {
     uint256 public constant THRESHOLD_NUMERATOR = 2;
     uint256 public constant THRESHOLD_DENOMINATOR = 3;
     uint256 public constant BLOCK_WAIT_PERIOD = 3;
+    uint256 public constant NUMBER_OF_BLOCKS_PER_EPOCH = 3;
+    uint256 public constant ERROR_AND_SAFETY_BUFFER = 3;
```

Comment:
What are those intended to be used for?

Add a comment?

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/442/commits/dcc5b1468ca5e3618b1c07254dbb501b44aae014